src/interface/src/styles/typography.scss

```diff
@@ -0,0 +1,125 @@
+// base
+body {
+  font-family: Public Sans, sans-serif;
```

Comment:
this is being applied, but we are overriding all over the place with roboto, so I dont think it ever gets to use this

Reply:
I also discovered this when I tried to change fonts in `body`, but maybe this is okay for now.
Should we have a convention to quote multi-word font names, or no?